spyder/plugins/updatemanager/widgets/update.py

```diff
@@ -612,7 +616,20 @@ def _start_updater(self):
         cmd = [updater_path, "--update-info-file", info_file]
         if self.restart_spyder:
             cmd.append("--start-spyder")
-        subprocess.Popen(" ".join(cmd), shell=True)
+
+        kwargs = dict(shell=True)
+        if os.name == "nt" and is_installed_all_users():
+            # Elevate UAC
+            kwargs.update(executable=find_program("powershell"))
+            cmd = [
+                "start",
+                "-FilePath", f'"{updater_path}"',
+                "-ArgumentList", ",".join([f"'{a}'" for a in cmd[1:]]),
+                "-WindowStyle", "Hidden",
+                "-Verb", "RunAs",
```

Comment:
```suggestion
                "-FilePath",
                f'"{updater_path}"',
                "-ArgumentList",
                ",".join([f"'{a}'" for a in cmd[1:]]),
                "-WindowStyle",
                "Hidden",
                "-Verb",
                "RunAs",
```

Parse this list as Black/Ruff would do it.